src/Components/Nav/Nav.tsx

```diff
@@ -4,20 +4,21 @@ import NavLink from "../NavLink/NavLink";
 import { default as NavLinkStyles } from "./Nav.module.scss";
 import bemCssModule from "bem-css-modules";
 import { useTranslation } from "react-i18next";
-import { Link } from "constants/types/Link";
+import { LinkElements } from "constants/types/Link";
+
 const style = bemCssModule(NavLinkStyles);
 
 const Nav = () => {
   const { t } = useTranslation();
 
-  const links: Link[] = [
+  const links: LinkElements[] = [
```

Comment:
Please try to be consistent with interfaces name. The There is array of string[] not strings[], number[] not numbers[].